src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -941,7 +941,8 @@ public void SarifTransformerTests_ToCurrent_NotificationExceptionWithStack()
                           },
                           ""region"": {
                             ""startLine"": 15,
-                            ""startColumn"": 9
+                            ""startColumn"": 9,
+                            ""endColumn"": 9
```

Comment:
Yes, that's right. In v1, if `endColumn` was absent, it meant an insertion point before `startColumn`. To express that in v2, you set `endColumn` == `startColumn`.